doc/Overview/Configuration/ConfigurationOverview.md

```diff
@@ -1,141 +1,158 @@
 ---
 uid: Overview.Configuration
 ---
-# Configuration and Settings
+# Configuration
 
-Uno.Extension.Configuration uses [Microsoft.Extensions.Configuration](https://www.nuget.org/packages/Microsoft.Extensions.Configuration) for any configuration related work.
+`Uno.Extensions.Configuration` provides a uniform way to read or write configuration data from a number of distinct sources. The implmentation of `IOptions<T>` from [Microsoft.Extensions.Options](https://docs.microsoft.com/dotnet/api/microsoft.extensions.options) allows for [read-only](https://docs.microsoft.com/dotnet/core/extensions/configuration#concepts-and-abstractions) access to values organized into **configuration sections**. The [writable configuration](xref:Learn.Tutorials.Configuration.HowToWritableConfiguration) pattern supports the ability to update configuration values at runtime.
 
-For more documentation on configuration, read the references listed at the bottom.
+This feature uses [Microsoft.Extensions.Configuration](https://www.nuget.org/packages/Microsoft.Extensions.Configuration) for any configuration related work. For more documentation on configuration, read the references listed at the bottom.
 
 ## AppSettings
 
-To use `appsettings.json` file packaged as `EmbeddedResource` in the application:  
+To use `appsettings.json` file packaged as **EmbeddedResource** in the application:  
 
 ```csharp
 private IHost Host { get; }
 
-public App()
+protected override void OnLaunched(LaunchActivatedEventArgs e)
 {
-    Host = UnoHost
-        .CreateDefaultBuilder()
-        .UseConfiguration(configure: builder => 
-            builder.EmbeddedSource<App>( includeEnvironmentSettings: true ))
-        .Build();
-    // ........ //
-}
-```  
+    var appBuilder = this.CreateBuilder(e)
+    .Configure(
+        host => host
+            .UseConfiguration(configure: builder => 
+                builder.EmbeddedSource<App>( includeEnvironmentSettings: true ))
+    );
+
+    Host = appBuilder.Build();
+...
+```
 
-To use `appsettings.json` file packaged as `Content` in the application:   
+To use `appsettings.json` file packaged as **Content** in the application:   
 
```

Comment:
Make a note that this isn't recommended, particularly if targeting wasm as data isn't from configuration won't be available immediately during startup